browser/ui/views/brave_help_bubble/brave_help_bubble_delegate_view.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_HELP_BUBBLE_BRAVE_HELP_BUBBLE_DELEGATE_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_HELP_BUBBLE_BRAVE_HELP_BUBBLE_DELEGATE_VIEW_H_
+
+#include <memory>
+#include <string>
+
+#include "chrome/browser/ui/browser.h"
+#include "chrome/browser/ui/views/bubble/webui_bubble_manager.h"
+#include "ui/base/metadata/metadata_header_macros.h"
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
+
+using views::View;
+using views::Widget;
+
+class BraveHelpBubbleDelegateView : public views::BubbleDialogDelegateView {
+ public:
+  METADATA_HEADER(BraveHelpBubbleDelegateView);
+
+  class Observer : public base::CheckedObserver {
+   public:
+    virtual void OnBubbleClosing(Widget* widget) {}
```

Comment:
Most examples in chromium's codebase doesn't seem to have a virtual dtor in case of base::CheckObserver. 

I see an override `~Observer() override {}`

Reply:
Oh, my bad. You're totally right as this one inherits base::CheckObserver. ~Observer() override = default could be added but I think it's good as is.